index.html

```diff
@@ -55,6 +55,42 @@ <h1>
             <a class="brand" href="">Let's get Start...</a>
           </p>
         </div>
+        <div class="container">
+          <h1>Avatar</h1>
+          <p>Here we created custome classes for different sizes of Avatar.</p>
+          <p>
+            Different sizes: avatar-sm, avatar-md, avatar-l, avatar-xl and In
+            img tag src attribute contain image path.
+          </p>
+          <div class="inline">
+            <div class="avatar-sm">
+              <img src="./assets/avatar.png" alt="avatar-small" />
+            </div>
+            <div class="avatar-md">
+              <img src="./assets/avatar.png" alt="avatar-medium" />
+            </div>
+            <div class="avatar-l">
```

Comment:
for large size avatar, you can add "avatar-lg" instead of "avatar-l". I will be more consistent with sm and md.